person.rb

```diff
@@ -27,6 +28,10 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  def add_rental(rental)
```

Comment:
This method should be adding rentals. For that, you need to initialize a new Rental like this `Rental.new()`. It should also be taking `book` and `date` as parameters and also pass the person as `self` as a parameter to itself to create the new rental, hence achieving the `has-many` association.